setup.py

```diff
@@ -24,7 +24,7 @@
     entry_points={"console_scripts": ["conda-vendor = conda_vendor.__main__:cli"]},
     install_requires=["ruamel.yaml", "conda-lock", "click"],
```

Comment:
If someone installs the environment via conda and then runs `pip install -e .` for development, the pip installed dependencies will conflict/overwrite the ones installed by conda.  Also, if someone only uses pip to install conda-vendor, some packages will be missing beacuse they are not in setup.py.  Even if the packages are added to setup.py, I don't think conda, mamba, and micromamba can be installed by pip, so this would be a problem.  Let's discuss this one.

Reply:
Lets definitely discuss outside of this PR.